src/styles/utils/zIndex.ts

```diff
@@ -0,0 +1,23 @@
+const zIndex = {
```

Comment:
Should we update the `zIndex` values in the theme and use those instead of private utils?

Reply:
Only concern is that apps are already using the default MUI `zIndex` and we may cause some breaking changes. We'd just need to check.

Reply:
Good catch! I missed the preexisting theme zIndex, so let me see if I can find it now...

Reply:
OK, I've moved zIndex to theme so we're using MUI's zIndicies as well as our new ones. Let me know if I need to improve any of those changes!